src/screens/Home/components/SignUp/index.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { useForm } from 'react-hook-form';
+import i18next from 'i18next';
+
+import WoloxLogo from '../../assets/LogoWolox-Original.png';
+
+import styles from './styles.module.scss';
+
+type InputsValues = {
+  user: {
+    email: string;
+    password: string;
+    passwordConfirmation: string;
+    firstName: string;
+    lastName: string;
+    locale: string;
+  };
+};
+
+function SignUp() {
+  const { register, handleSubmit } = useForm<InputsValues>();
+  const onSubmit = (data: InputsValues) => console.log(data);
+
+  return (
+    <form className={styles.signUpForm} onSubmit={handleSubmit(onSubmit)}>
+      <img src={WoloxLogo} className={`m-bottom-3 ${styles.logo}`} />
+      <div className={`column start m-bottom-3 ${styles.inputContainer}`}>
+        <label htmlFor="user.firstName" className={`m-bottom-2 ${styles.label}`}>
+          {i18next.t('SignUp:firstName')}
+        </label>
+        <input {...register('user.firstName')} className={styles.input} />
```

Comment:
Since you're repeating these `user.x` strings, it's better to move them to an enum:

```ts
enum UserFieldIds {
  FIRST_NAME = 'user.firstName',
  ...
}
```

Then use the enum values